npm/npm.go

```diff
@@ -280,13 +288,19 @@ func NewNetworkPolicyManager(clientset *kubernetes.Clientset, informerFactory in
 }
 
 // reconcileChains checks for ordering of AZURE-NPM chain in FORWARD chain periodically.
-func (npMgr *NetworkPolicyManager) reconcileChains() error {
+func (npMgr *NetworkPolicyManager) reconcileChains(stopCh <-chan struct{}) {
 	iptMgr := iptm.NewIptablesManager()
-	select {
-	case <-time.After(reconcileChainTimeInMinutes * time.Minute):
-		if err := iptMgr.CheckAndAddForwardChain(); err != nil {
-			return err
+
```

Comment:
you can use newlines to separate logical blocks, otherwise don't use extra vertical whitespace